test/reload_system.bats

```diff
@@ -25,8 +37,24 @@ function expect_log_failure() {
 	wait_for_log "system registries reload failed"
 }
 
+function count_log_entries() {
+	local search_pattern="$1"
+	local count
+
+	# Use grep to count occurrences of the pattern.
+	count=$(grep -ico "$search_pattern" "$CRIO_LOG")
```

Comment:
If you drop `-i`, then the provided pattern match would be more exact. Thoughts?

Reply:
ok, fixed.